examples/agents/in-house-ctf/README.md

```diff
@@ -0,0 +1,57 @@
+# In House CTF Tasks
+This directory includes Capture The Flag challenges migrated from Google DeepMind's [Dangerous Capabilities Evaluations](https://github.com/google-deepmind/dangerous-capability-evaluations/tree/main/in_house_ctf) (originally published in [Evaluating Frontier Models for Dangerous Capabilities](https://arxiv.org/pdf/2403.13793)). Here we include 9 of the 13 In-House CTF challenges (excluding 4 challenges that have faulty Docker images from the original repository). 
+
+This implementation uses [tool calling](https://inspect.ai-safety-institute.org.uk/tools.html) to enable the model to execute bash commands. The flag values has been replaced with placeholders `REPLACE_ME_WITH_RANDOM_FLAG_N` to prevent them from ending up in scrapable GitHub data, the actual values will be updated during evaluation run.
```

Comment:
Are we still doing this or relying on the flags already in the pre-built images?

Reply:
We're using the flags in the pre-built images, I kept them as a record for the flags, but I guess we can just remove them altogether?

Reply:
I think we should just be clear that the Dockerfile is there as documentation and in case the user wants to build their own images (and in the case that they want to build their own images we should let them know they should replace w/ random flags).